cli/docs/flags.go

```diff
@@ -43,6 +43,14 @@ const (
 	Poetry = "poetry"
 )
 
+const (
+	Sca                  = "sca"
+	Iac                  = "iac"
+	Sast                 = "sast"
+	Secrets              = "secrets"
+	NoContextualAnalysis = "without-contextual-analysis"
```

Comment:
Internally I'd rename it to WithoutCA - just to keep it all a bit cleaner - but that's a personal preference - your call on that one.